src/Microsoft.Extensions.Configuration.UserSecrets/PathHelper.cs

```diff
@@ -3,13 +3,31 @@
 
 using System;
 using System.IO;
+using Microsoft.Extensions.FileProviders;
+using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 
 namespace Microsoft.Extensions.Configuration.UserSecrets
 {
     public class PathHelper
     {
         internal const string Secrets_File_Name = "secrets.json";
+        internal const string Config_File_Name = "project.json";
+
+        public static string GetSecretsPath(IFileProvider provider)
+        {
+            var fileInfo = provider.GetFileInfo(Config_File_Name);
```

Comment:
Yeah, I was looking at the PhysicalFileProvider which returns a NotFoundFileInfo, but other FileProviders could return null